.github/dependabot.yml

```diff
@@ -63,6 +60,27 @@ updates:
       - dependency-name: 'reflux'
       - dependency-name: 'sprintf-js'
       - dependency-name: 'u2f-api'
+  # For Sentry SDK updates, we do not want to limit the open PRs
+  - package-ecosystem: npm
+    open-pull-requests-limit: 0
```

Comment:
I don't think it works like this -- last we tried you can only have one `npm` section in a dependabot config

Reply:
ah damn, that sucks - I thought that would work :( does that mean it is not possible to realize this? 😬 
What are thoughts about increasing the PR limit (10), or finding a different way to deal with those? As of now dependabot seems basically blocked because of open PRs with low prio 🤔

Reply:
afaik for packages we care about we've just been doing them periodically on our own.  dependabot isn't very good at bumping packages anyway (lol I know right?) so it's been smoother when a person does it

Reply:
Yeah I mean this is what we kind of did so far, but we always lag behind a bit with the SDK and it would be nice to be up-to-date there, as it is our premier dogfooding app as well 🤔

Reply:
Yeah doesn't seem to be supported: https://github.com/dependabot/dependabot-core/issues/1778

Though looks like [there's a workaround](https://github.com/dependabot/dependabot-core/issues/1778#issuecomment-1988140219) if you have one config set `target-branch: master`